### ISSUE TITLE
Wiki pages updates

### ISSUE THREAD

**Issue by Gizmokid2005** [wdehoog/sailfish-bacon-images#4]
Github doesn't allow PRs for wiki pages. I just went through flashing Sailfish (THANK YOU!) on my OPO today and was doing my research and found a way to make the main wiki page better at least, I've attached it. Maybe it'll help others (or could use some updates before being used because I did something wrong?) in the future.
[Home.txt](https://github.com/wdehoog/sailfish-bacon-images/files/1116736/Home.txt)

**Comment by wdehoog** [comment]
Thanks for your request. I guess you wanted me to update the name of the CM12.1 image to use as indeed it was wrong. The text now mentions the other name and points to the wiki page.

**Comment by Gizmokid2005** [comment]
Yeah, you linked to your own image in the snapshots which also addressed the issues with the sensors making that section also unneeded. Just helping to clean up as much as I can :)
